src/nunit.analyzers/nunit.analyzers.csproj

```diff
@@ -15,27 +15,19 @@
   </PropertyGroup>
 
   <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'" >
-    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="3.8.0" />
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="4.0.1" />
```

Comment:
Wouldn't this affect the version of Visual Studio that can use the analyzer? I thought that Roslyn 3.x mean VS 2019, so I would assume that version 4.x means VS 2022 ?